test/tests/functional/pbs_dup_acc_log_for_resv.py

```diff
@@ -51,6 +52,52 @@ def setUp(self):
             self.skipTest('test requires two MoMs as input, ' +
                           'use -p moms=<mom1>:<mom2>')
 
+    def check_log_resv_time(self, accounting_log_msg, start_tup,
+                            end_tup, check_status):
+        """
+        This is a helper function to reterive start and end time
+        of standing rervation from accounting log.
+        """
+        find_start = m[start_tup][end_tup].find("start=")
```

Comment:
This function assumes an ordering of fields in the accounting 'B' record.  The only guaranteed thing we know about the accounting records is they are space delaminated.  You should split the record on a space and then find the start/end record and then split it on '=''.